github-actions/check-data-assets/action.yml

```diff
@@ -76,7 +76,9 @@ runs:
         if [[ "$PR_NUMBER" != "" ]]; then
           # Don't fail the comment step (yet) if we get a BLOCK result, but capture the exit code
           set +e
-          COMMENT_MARKDOWN=$("$PYTHON_PATH" -m gable.cli data-asset check $DEBUG --output markdown ${{ inputs.data-asset-options }} 2> debug_output.txt)
+          VENV_BIN_PATH=$(python -c "import sys; print(sys.prefix + '/bin')")
+          GABLE_API_ENDPOINT="${{ inputs.gable-api-endpoint }}" GABLE_API_KEY="${{ inputs.gable-api-key }}" \
```

Comment:
I think these should already be set by the
```
export GABLE_API_ENDPOINT="${{ inputs.gable-api-endpoint }}"
export GABLE_API_KEY="${{ inputs.gable-api-key }}"
```
above - was it not working with your change?